src/style.css

```
@@ -0,0 +1,114 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.form-search {
+  margin-top: 20px;
+  position: relative;
+}
+
+.search-icon-big {
+  border: none;
+  display: block;
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  background-color: transparent;
+}
+
+.search-icon-small {
+  border: none;
+  display: block;
+  position: absolute;
+  background-color: transparent;
+  left: 10px;
+  top: 10px;
+}
```

You can create default styles for **search-icon** and add modifier like **search-icon-small** and add there styles that applies only for small icon.

```suggestion
.search-icon {
  border: none;
  display: block;
  position: absolute;
  left: 26px;
  top: 25px;
  background-color: transparent;
}

.search-icon-small {
  left: 10px;
  top: 10px;
}
```

- [ ] combine classes for common layput
- [ ] adjust search-icon-small position to be aligned with the designs
- [ ] add proper class names in HTML - element can have more than one class